manifests/master/pdf_stack.pp

```diff
@@ -1,7 +1,7 @@
 # Dependencies for PDF rendering
 class classroom::master::pdf_stack {
 
-  $enabled = $classroom::offline ? {
+  $enabled = defined('$classroom::offline') and $classroom::offline ? {
```

Comment:
@binford2k @js Can you tell if this is an OK change to make? I did it to get the rspec tests working. Otherwise, they would fail with a "$classroom::offline variable is undefined". I tried to define the variable in the pre_conditions clause in the spec test file, but I could not do that either because it complained about setting a variable in a different namespace.

Thoughts?